src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -283,14 +283,17 @@ export default {
 			:disabled="disabled"
 			:type="inputType"
 			:value="value"
-			v-bind="inputProps"
+			:checked="isChecked"
```

Comment:
A button also cannot be checked :see_no_evil:

Reply:
@Pytal

Reply:
This `input` is rendered only for non-buttons. The root element itself is a `button` when it has `type="button"`

Reply:
A bit confusing =D

Reply:
Ah right, missed the line 278
https://github.com/nextcloud-libraries/nextcloud-vue/blob/383dfa00376a7bdfe2bffc2358ed4e0d59ff17d1/src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue#L278